tests/utils/test_libxc.py

```diff
@@ -0,0 +1,188 @@
+    ('M08-SO', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_M08_SO', 'XC_HYB_MGGA_C_M08_SO']),
+    ('M11', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_M11', 'XC_HYB_MGGA_C_M11']),
+    ('MN15', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_MN15', 'XC_HYB_MGGA_C_MN15']),
+    ('SCAN0', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_SCAN0', 'XC_MGGA_C_SCAN']),
+    ('ωB97X-D', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_WB97X_D']),
+    ('ωB97X-V', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_WB97X_V', 'XC_NLC_XC_VV10']),
+    ('ωB97M-V', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_WB97M_V', 'XC_NLC_XC_VV10']),
+    # ('revM06-L (hyb-like)', 'hybrid-meta-GGA', ['XC_HYB_MGGA_X_M06', 'XC_MGGA_C_M06_L']), #this is a hack, not a real hybrid!
+    # ----------- meta-GGA + NL combos ----
+    (
+        'SCAN + rVV10',
+        'meta-GGA',
+        ['XC_MGGA_X_SCAN', 'XC_NLC_XC_RVV10', 'XC_NLCF_XC_RVV10'],
+    ),
+    (
+        'r2SCAN + VV10',
+        'meta-GGA',
+        ['XC_MGGA_X_R2SCAN', 'XC_NLC_XC_VV10', 'XC_NLCF_XC_VV10'],
+    ),
+    ('B97M-V (explicit)', 'meta-GGA', ['XC_MGGA_X_B97M_V', 'XC_NLC_XC_VV10']),
+    ('B97M-rV (explicit)', 'meta-GGA', ['XC_MGGA_X_B97M_RV', 'XC_NLC_XC_RVV10']),
+    ('TPSS + D3(BJ)', 'meta-GGA', ['XC_MGGA_X_TPSS', 'XC_MGGA_C_TPSS']),
+    ('SCAN + D3(BJ)', 'meta-GGA', ['XC_MGGA_X_SCAN', 'XC_MGGA_C_SCAN']),
+    ('r2SCAN + D4', 'meta-GGA', ['XC_MGGA_X_R2SCAN', 'XC_MGGA_C_R2SCAN']),
+    # ------------- GGA + NL combos --------
+    ('PBE + VV10', 'GGA', ['XC_GGA_X_PBE', 'XC_NLC_XC_VV10']),
+    ('PBE + rVV10', 'GGA', ['XC_GGA_X_PBE', 'XC_NLC_XC_RVV10']),
+    ('revPBE + VV10', 'GGA', ['XC_GGA_X_RPBE', 'XC_NLC_XC_VV10']),
+    ('BLYP + VV10', 'GGA', ['XC_GGA_X_B88', 'XC_GGA_C_LYP', 'XC_NLC_XC_VV10']),
+    ('PBE + D3(BJ)', 'GGA', ['XC_GGA_X_PBE', 'XC_GGA_C_PBE']),
+    ('PBEsol + D3', 'GGA', ['XC_GGA_X_PBE_SOL', 'XC_GGA_C_PBE_SOL']),
+    ('BP86 + D3', 'GGA', ['XC_GGA_X_B88', 'XC_GGA_C_P86']),
+    # ------------- Hybrids + NL -----------
+    ('B3LYP + D3(BJ)', 'hybrid-GGA', ['XC_HYB_GGA_XC_B3LYP']),
+    ('PBE0 + D3(BJ)', 'hybrid-GGA', ['XC_HYB_GGA_XC_PBE0', 'XC_HYB_GGA_XC_PBEH']),
+    ('HSE06 + D3(BJ)', 'hybrid-GGA', ['XC_HYB_GGA_XC_HSE06']),
+    (
+        'ωB97X-V (explicit)',
+        'hybrid-meta-GGA',
+        ['XC_HYB_MGGA_X_WB97X_V', 'XC_NLC_XC_VV10'],
+    ),
+    (
+        'ωB97M-V (explicit)',
+        'hybrid-meta-GGA',
+        ['XC_HYB_MGGA_X_WB97M_V', 'XC_NLC_XC_VV10'],
+    ),
+    # -------- a few “code nicknames” ------
+    ('PBE0 (alias PBEh)', 'hybrid-GGA', ['XC_HYB_GGA_XC_PBEH', 'XC_HYB_GGA_XC_PBE0']),
+    ('HSE (generic)', 'hybrid-GGA', ['XC_HYB_GGA_XC_HSE06', 'XC_HYB_GGA_XC_HSE03']),
+    ('B3PW', 'hybrid-GGA', ['XC_HYB_GGA_XC_B3PW91']),
+]
+
+
+@pytest.mark.parametrize('raw, expected_family, labels', _COMMON_XC_CASES)
+def test_dft_expands_100_common_functionals(raw, expected_family, labels):
+    dft = DFT()
+    dft.xc = XCFunctional(raw_string=raw)
+    dft.normalize(EntryArchive(), logger=logger)
```

Comment:
This is a test of the schema, not the utilities. Pls move to the other testing file.

Reply:
You mean like the output from `expand_to_libxc_labels`?
That's alright, but I would then call the method directly, without the `normalize` wrapper.